embassy/src/time/mod.rs

```diff
@@ -1,3 +1,6 @@
+/// Time abstractions
+/// To use these abstractions, first call `set_clock` with an instance of an monotonic `Clock`.
```

Comment:
typo: a monotonic

Just saying "an instance of Clock" would be enough. The Clock docs already specify it.